src/Contexts/SingleItemContext.php

```diff
@@ -55,6 +55,10 @@ public function init($params)
         $this->setComponents = $params['setComponents'];
         $this->setAttributeMap = $params['setAttributeMap'];
         $this->requestedVariationUrl = explode('?', $this->request->getUri())[0];
+
+        $metaTitle = $this->webstoreConfig->urlTitleItemName - 1;
```

Comment:
Ich würde im Template direkt auf die WebstoreConfig zugreifen, anstatt hier ein weiteres Feld in den Context zu schreiben.